api/src/main/java/org/pmiops/workbench/google/CloudStorageServiceImpl.java

```diff
@@ -36,7 +36,7 @@ public CloudStorageServiceImpl(Provider<WorkbenchConfig> configProvider) {
 
   @Override
   public String readInvitationKey() {
-    return readCredentialsBucketString("invitation-key.txt");
+    return getCredentialsBucketString("invitation-key.txt");
```

Comment:
I'd probably make an enum with these values or store them in per-env configs. In principle we could have separate Moodles, etc.